### PR TITLE
update ics-openvpn

### PR DESCRIPTION
**Pull request by fkooman**
update ics-openvpn for #320 

also update build script/builder setup

**NOTE**: the .gitmodules points to /fkooman/ and not /eduvpn/ !

### CODE REVIEW

**Comment by dzolnai** [comment]
> NOTE: the .gitmodules points to /fkooman/ and not /eduvpn/ !

Why can't it point to /eduvpn/? I merged the update.

**Comment by fkooman** [comment]
> > NOTE: the .gitmodules points to /fkooman/ and not /eduvpn/ !
> 
> Why can't it point to /eduvpn/? I merged the update.

It was not merged, only approved. I now merged it. Will update this PR.

**Comment by fkooman** [comment]
Only modifying `.gitmodules` is enough, right?

**Comment by dzolnai** [comment]
No, you need to update the commit reference:
```
cd ics-openvpn/
git fetch eduvpn #(not sure what this remote is called, might be origin)
git reset --hard eduvpn/master
cd ..
git add --all
git commit -m "Update ics-openvpn commit"
```

**Comment by fkooman** [comment]
```
[fkooman@fralen-tuxed-net eduvpn-android]$ git submodule sync
Synchronizing submodule url for 'appauth'
Synchronizing submodule url for 'ics-openvpn'
[fkooman@fralen-tuxed-net eduvpn-android]$ cd ics-openvpn/
[fkooman@fralen-tuxed-net ics-openvpn]$ git remote -v
origin	https://github.com/eduvpn/ics-openvpn (fetch)
origin	https://github.com/eduvpn/ics-openvpn (push)
[fkooman@fralen-tuxed-net ics-openvpn]$ git pull
Already up to date.
[fkooman@fralen-tuxed-net ics-openvpn]$ cd ..
[fkooman@fralen-tuxed-net eduvpn-android]$ git commit -a -m 'point to eduvpn/ics-openvpn:master'
[master 3adb810] point to eduvpn/ics-openvpn:master
 1 file changed, 1 insertion(+), 1 deletion(-)
[fkooman@fralen-tuxed-net eduvpn-android]$ git push
Enumerating objects: 3, done.
Counting objects: 100% (3/3), done.
Delta compression using up to 4 threads
Compressing objects: 100% (2/2), done.
Writing objects: 100% (2/2), 276 bytes | 276.00 KiB/s, done.
Total 2 (delta 1), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
To github.com:fkooman/eduvpn-android.git
   a46d225..3adb810  master -> master
[fkooman@fralen-tuxed-net eduvpn-android]$ 
```

**Comment by dzolnai** [comment]
I can see the update, looks good.

File: app/build.gradle [hunk 1]
```diff
@@ -14,7 +14,7 @@ android {
         targetSdkVersion 29
         versionCode 15
         versionName "2.0.2"
-        ndkVersion "20.0.5594570"
+        ndkVersion "21.0.6113669"
```
Comment (dzolnai) [inline app/build.gradle:17]: This will break Buildozer, because it only has one NDK version available. Not sure if you are OK with that.

**Review by dzolnai — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (fkooman) [reply app/build.gradle:17]: This 21.0.6113669 is also what is used in ics-openvpn, so I just used the same version here... or did I make a typo?

Comment (dzolnai) [reply app/build.gradle:17]: I think Buildozer looks at the one on the top level. We can try, maybe you are right :)

**Comment by fkooman** [comment]
Thanks for the reviewing!

File: app/build.gradle (hunk 1 above)
Comment (fkooman) [reply app/build.gradle:17]: 102: > No version of NDK matched the requested version 21.0.6113669. Versions available locally: 20.0.5594570, 21.1.6352462

Aha... but ics-openvpn uses 21.0.6113669, so I'd either have to also modify that, or hmm?

**Pull request merged**